generate.sh

```diff
@@ -35,6 +37,7 @@ usage() {
     echo "  --dest <pathToDestRepo>            A path to the root of the repo to copy source into."
     echo "  --type <packageType>               Type of the package to generate. Accepted values: ref (default) | text."
     echo "  --feeds <nugetFeeds>               A semicolon-separated list of additional NuGet feeds to use during restore."
+    echo "  --genapi-backend                   The GenAPI backend used to generate reference assemblies [cci|roslyn]. Default is '$backend'."
```

Comment:
The pattern used to indicate the accepted and default values is inconsistent with the `type` option.  The script should use a consistent pattern.  I personally don't care which pattern is used.

Reply:
Updated the description of the `--genapi-backend` option